test/cucumber/steps/general.js

```diff
@@ -59,7 +59,7 @@ module.exports = function() {
   });
 
   // Vocabulary proxy over the previous action for better scenarios readability.
-  this.When(/^I call "([^"]*)"$/, function(_command) {
+  this.When(/^I call "gavel.validate(([^"]*))"$/, function(_, _command) {
```

Comment:
Shouldn't it be `gavel.validate\(([^\)]*)\)`? Not important if it works correctly like this

Reply:
It does seem to work right now, so we can improve it in #113 :)